seed/models/columns.py

```diff
@@ -139,6 +137,7 @@ class Column(models.Model):
         'DateTimeField': 'datetime',
         'BooleanField': 'boolean',
         'JSONField': 'string',
+        'PolygonField': 'geometry',
```

Comment:
Note that I introduced a new `data_type`. I believe this didn't negatively impact areas of the codebase outside of the work that I was doing, but I think this is important to note